src/main/java/com/google/api/codegen/util/php/PhpTypeTable.java

```diff
@@ -105,7 +105,7 @@ public String getAndSaveNicknameForInnerType(
   /**
    * A set of PHP keywords and built-ins. keywords: http://php.net/manual/en/reserved.keywords.php
    */
-  private static final ImmutableSet<String> KEYWORD_BUILT_IN_SET =
+  public static final ImmutableSet<String> KEYWORD_BUILT_IN_SET =
```

Comment:
Could you move all of the `KEYWORD_BUILT_IN_SET` constants to the corresponding `{Lang}NameFormatter` classes? It is a more appropriate location, and I don't think any of them are referenced within the corresponding `{Lang}TypeTable` classes.